tests/test_plotting.py

```diff
@@ -58,6 +58,7 @@ def test_init_plotscript(default_conf, mocker, testdatadir):
     assert "ohlcv" in ret
     assert "TRX/BTC" in ret["ohlcv"]
     assert "ADA/BTC" in ret["ohlcv"]
+    assert default_conf['strategy'].bot_started is True
```

Comment:
I get why this is failing, but I'm not sure how I would test this out for plotting

Reply:
i don't think you can reliably test this - not the way plotting is setup (the strategy object is a local object in that context).

You could try to mock or patch the method and see if it's been called (that's what you're really after) - but i'm not sure that'll work with the strategy loader correctly.
it's most likely not worth the effort though - we don't have to have this tested for all cases i think.